docs/LEARNING.md

```diff
@@ -1,13 +1,22 @@
 # Learning
 
-<!-- TODO: write document
+## WebAssembly Text Format
 
-  This document should link to learning resources.
+Most WebAssembly resources on the web focus on high level concepts and jump immediately to building a WebAssembly module in a higher level language. Here are a handful of resources that cover the Text Format as used in this track.
 
-  Feel free to link to any good learning resources you know, whether they
-  be websites, blogs, books, videos or courses.
+### WAT Websites
```

Comment:
I'd simplify this to just:

```suggestion
### Websites
```

as we're already in the context of WebAssembly Text Format